src/encoded/static/components/search.js

```diff
@@ -1,27 +1,33 @@
-import React from 'react';
-import cloneWithProps from 'react/lib/cloneWithProps';
-import cx from 'react/lib/cx';
-import url from 'url';
-import _ from 'underscore';
-import queryString from 'query-string';
-import { DropdownButton } from '../libs/bootstrap/button';
+'use strict';
+var React = require('react');
```

Comment:
That’s fine for now. Master took search.js to ES6 after 4755-react-15 branched off, and the merges from master kept the 4755-react-15 code as much as possible.